tst-cond1.c

```diff
@@ -85,13 +85,15 @@ static int do_test (void)
 	}
```

Comment:
Can you provide some more context about where you're going with this? Why change this specific test instead of add another test - and why 2 instead of 20... Needs some detail so I can understand your intent.